tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/sealed_array_semantics_negative.bal

```diff
@@ -113,3 +113,24 @@ function openToSealArrrayAssignment() {
     int[] intArr1 = [];
     int[*] intArr2 = intArr1.clone();
 }
+
```

Comment:
I believe we need to consider 
```Ballerina
type ARR int[*];
```

Although I believe example 2 is okay
```Ballerina
// Example 2
function value(int[*] x = []) {}
```
@chiranSachintha Cant we think of the example 2 as having zero members